docs/developer-guide/sinks-guide.rst

```diff
@@ -0,0 +1,122 @@
+How to implement a custom Robusta sink
```

Comment:
Can we rename the title to "Implementing a custom sink" as that matches the style of the other page titles?
  
![Screen Shot 2022-08-28 at 0 10 33](https://user-images.githubusercontent.com/494087/187048082-75c1011f-7d45-4e62-8057-b16ecc88a6ea.png)